ledger-api/grpc-definitions/com/daml/ledger/api/v2/package.proto

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v2;
+
+// Ledger API version V2
```

Comment:
@nmarton-da: another small consideration: we did not bump the Ledger API upon the Daml 2.0 release.

However, releasing Daml 3.0 with Ledger API v2 seems prone to confusion. It might be easier to explain why there is no v2 version, and just move all of the new services to v3.

Reply:
I like the idea to align with the daml major version :+1: 
@rgugliel-da @cocreature WDYT?
This would essentially mean moving here stuff to v3 package, change comments etc.

Reply:
don't have strong feelings there, I'm happy with v3

Reply:
I am used to have discrepancies so I don't see a use case to change but why not.

Reply:
We may not need an upgrade for Daml 4 just like we didn't need one going from Daml 1 to Daml 2, so we might end up making a bunch of unnecessary changes to the package structure of generated code without a real need for it. I don't think there's any downside in having the Ledger API being versioned at a separate pace. Daml-LF and the Canton Protocol do so as well. They all converge in the SDK version.

Reply:
I also like the API local version, and agree that technically we are already deviating form the major SDK version. Let's keep it v2 for now, and I will point this out to product in the next review round, if they think there is a compelling reason to change we could still do it, should be an easy change.
Thanks for your thoughts on this!